app/components/nav-rail-pinned-item.tsx

```
@@ -28,17 +28,26 @@ export default function NavRailPinnedItem(
-                <md-ripple />
-                <div className={`nav-item-content flex extended-gap-${props.extended} items-center`}>
-                    <div className="cover flex items-center">
-                        <Image src={props.imgSrc} alt={props.text} width={props.width} height={props.width} />
-                    </div>
-                    <div className={`label-container text-ellipsis items-center extended-${props.extended}`}>
-                        <p className={`label text-center text-ellipsis extended-${props.extended}`}>{props.text}</p>
-                    </div>
-                </div>
+        <div className={`nav-item rounded-full cursor-pointer font-medium selected-${props.selected}`} role='link' onClick={props.onClick}>
+            <Link className={`state-layer relative flex extended-gap-${props.extended} items-center justify-between rounded-full padding-${props.extended ? "extended" : "collapsed"}`} href={props.href!}>
+                <md-ripple></md-ripple>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)